packages/experiments/src/components/Text/TextPage.tsx

```diff
@@ -19,7 +19,7 @@ export class TextPage extends React.Component<IComponentDemoPageProps, {}> {
         }
         propertiesTables={
           <PropertiesTableSet
-            sources={[require<string>('!raw-loader!@uifabric/experiments/src/components/Text/Text.tsx')]}
+            sources={[require<string>('!raw-loader!@uifabric/experiments/src/components/Text/Text.types.tsx')]}
```

Comment:
I know `Text.types.tsx` wasn't created in this PR but we seem to be naming a lot of files with `.tsx` that have no JSX in them.